app/assets/javascripts/models/activity.js

```diff
@@ -1,44 +1,47 @@
-import SharedModelMethods from '../mixins/shared_model_methods';
+import SharedModelMethods from "../mixins/shared_model_methods";
 
 const Activity = Backbone.Model.extend({
   defaults: {
-    name: 'activity',
-    date: '',
-    action: '',
-    subject_changes: '',
+    name: "activity",
+    date: "",
+    action: "",
+    subject_changes: "",
   },
 
-  name: 'activity',
+  name: "activity",
 
-  i18nScope: 'activerecord.attributes.',
+  i18nScope: "activerecord.attributes.",
 
-  timestampFormat: 'd mmm yyyy',
+  timestampFormat: "d mmm yyyy",
 
-  initialize: function(args) {
+  initialize: function (args) {
     var data = args.activity;
 
     this.i18nScope += data.subject_type.toLowerCase();
     this.set({
       date: new Date(data.updated_at).format(this.timestampFormat),
       action: this.humanActionName(data.action),
-      subject_changes: this.parseChanges(data.subject_changes)
+      subject_changes: this.parseChanges(data.subject_changes),
     });
   },
 
-  humanActionName: function(action) {
-    return I18n.t(action, {scope: 'activity.actions'});
+  humanActionName: function (action) {
+    return I18n.t(action, { scope: "activity.actions" });
   },
 
-  parseChanges: function(changes) {
-    return _.map(changes, function(value, key) {
-      if (key === 'documents_attributes') key = 'documents';
```

Comment:
Was this `documents_attributes` related to the Story Attachment? I mean, when the key was `documents_attributes` is that because we were changing the Story Attachment? Not that this is wrong I am just trying to get context about that

Reply:
Btw, I notice this `key` in this file:
https://github.com/Codeminer42/cm42-central/blob/9276421e0f87b792b2dddd0ce77154254a7b23e4/app/presenters/activity_presenter.rb#L58
Do we still need this line ^^?

Reply:
I think so since the stories have an attribute called `documents_attributes` and the only operation that implements something related to document changes is at the `StoriesOperations`

Reply:
By seeing this, I realized that may be better to leave those things at the presentation layer the way that they were.
Cause I think that what this does is take all the Activities in the db and parse them to show at the project home page.
If we delete this, and leave the records of document changes activities in the db, it may cause errors.

Reply:
What do you think?

Reply:
Hum... that's a good point. Since I don't have a lot of context about the project, I think I am not the best person to answer this, but I think that's worth testing. Maybe you can request access to Cloudinary, add some attachments locally on the master branch to simulate a real user/project that uses that feature, and then change to this branch and check if there are errors. I can try to help if you need to, just let me know.